src/metabase/driver/generic_sql.clj

```diff
@@ -312,7 +308,7 @@
     (url-percentage url-count total-count)))
 
 
-(defn fast-field-percent-urls
+#_(defn fast-field-percent-urls
   "Fast, default implementation of `field-percent-urls` that avoids a full table scan."
   [driver field]
   (let [table       (field/table field)
```

Comment:
all these things that are commented out, is that because they're no longer used? If so, we should probably just delete the lines instead